frontend/app/src/components/StatusWidget/StatusWidget.test.tsx

```diff
@@ -354,7 +372,9 @@ describe("Running Icon", () => {
 
     vi.runAllTimers()
 
-    icon = screen.queryByRole("img")
-    expect(icon).toHaveAttribute("src", "/src/assets/img/icon_running.gif")
+    await waitFor(() => {
+      icon = screen.queryByRole("img")
```

Comment:
nit: since `icon` is not used again outside of the function, might be nicer to write `const icon = ...` and make the other `icon` variable above also `const`